File: app/src/main/java/com/example/exam_1/ResultActivity.java
package com.example.exam_1;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class ResultActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_result);

        double num1 = getIntent().getDoubleExtra("num1", 0.0);
        double num2 = getIntent().getDoubleExtra("num2", 0.0);
        String result = getIntent().getStringExtra("result");
        String operation = getIntent().getStringExtra("operation");

        if (result == null) result = "";
        if (operation == null) operation = "";

        TextView tvResultDetail = findViewById(R.id.tvResultDetail);
        tvResultDetail.setText("Num1: " + num1 + "\nNum2: " + num2 + "\nOperation: " + operation + "\nResult: " + result);
    }
}

File: app/src/main/java/com/example/exam_1/ResultActivity.kt
package com.example.exam_1

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class ResultActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_result)

        val num1 = intent.getDoubleExtra("num1", 0.0)
        val num2 = intent.getDoubleExtra("num2", 0.0)
        val result = intent.getStringExtra("result") ?: ""
        val operation = intent.getStringExtra("operation") ?: ""

        val tvResultDetail = findViewById<TextView>(R.id.tvResultDetail)
        tvResultDetail.text = "Num1: $num1\nNum2: $num2\nOperation: $operation\nResult: $result"
    }
}
